.editorconfig

```diff
@@ -3,13 +3,7 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 4
+indent_size = 2
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
-
-[*.md]
-trim_trailing_whitespace = false
```

Comment:
Just FYI, trailing whitespace is significant in Markdown files, it's used for line breaks.

Reply:
Thanks @emackey, I did not know that.  Prettier actually does the right thing here, I put it back so that editors that don't know about MD specifically don't conflict with our formatting.

782b6065f0ec43e7840c7877d1203efdf9ced89e in master